packages/explorer/src/explorer.controller.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/explorer
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {inject} from '@loopback/context';
+import {
+  RestBindings,
+  RestServerConfig,
+  OpenApiSpecForm,
+  Request,
+  Response,
+} from '@loopback/rest';
+import * as ejs from 'ejs';
+import * as fs from 'fs';
+import * as path from 'path';
+
+// TODO(bajtos) Allow users to customize the template
```

Comment:
I'll add a short note to the README. 

FWIW, to customize the UI look, I think users may need to tweak more than just the index html file, e.g. provide a custom css file instead of the default `swagger-ui.css`.

Let me open a new GH issue to keep track of this missing feature: https://github.com/strongloop/loopback-next/issues/2023